src/Microsoft.ML.Core/EntryPoints/ComponentFactory.cs

```diff
@@ -29,6 +29,25 @@ public interface IComponentFactory<out TComponent>: IComponentFactory
         TComponent CreateComponent(IHostEnvironment env);
     }
 
+    /// <summary>
+    /// A class for creating a component with no extra parameters (other than an <see cref="IHostEnvironment"/>)
+    /// that simply wraps a delegate which creates the component.
+    /// </summary>
+    public class SimpleComponentFactory<TComponent> : IComponentFactory<TComponent>
```

Comment:
>public class SimpleComponentFactory [](start = 4, length = 35)

This can be sealed I hope?

Reply:
fixed.

---
In reply to: [210433599](https://github.com/dotnet/machinelearning/pull/681#discussion_r210433599) [](ancestors = 210433599)